docs/DeploymentGuide.md

```diff
@@ -41,9 +41,44 @@ In order to deploy OHDSI on Azure, you will need the following prerequisites:
 ## Data platform
 
 As noted above this solution currently supports storing the CDM on a managed PostgresSQL and on Azure Synapse Dedicated Pool. Please choose the right SKUs for each of the platforms depending on your dataset size.
-Note that when using the Synapse option, a default table distibution is used. If you intent to use a large dataset that might to be adjusted per the comments [here](https://github.com/OHDSI/CommonDataModel/blob/main/inst/ddl/5.4/synapse/OMOPCDM_synapse_5.4_ddl.sql) (although this quickstart doesn't support this out-of-the-box you can make the right moifications yourself).
+Note that when using the Synapse option, a default table distribution is used. If you intent to use a large dataset that might to be adjusted per the comments [here](https://github.com/OHDSI/CommonDataModel/blob/main/inst/ddl/5.4/synapse/OMOPCDM_synapse_5.4_ddl.sql) (although this quickstart doesn't support this out-of-the-box you can make the right modifications yourself).
 
 ## Permissions
 
 All the non-admin users you define in the deployment will automatically get permission to the data source created. 
-If you add more users and/or new data sources then you should use Atlas to grant additional permissions.
\ No newline at end of file
+If you add more users and/or new data sources then you should use Atlas to grant additional permissions.
+
+## How to access Atlas
+
+After deploying your OHDSI environment you will have the following resources on your Azure subscription:
+
+| TYPE                                            | NAME                  |
+|-------------------------------------------------|-----------------------|
+| App Service                                     | app-ohdsiatlas-myenv  |
+| App Service                                     | app-ohdsiwebapi-myenv |
+| App Service plan                                | asp-myenv             |
+| Managed Identity                                | id-ohdsiwebapi-myenv  |
+| Key vault                                       | kv-myenv              |
+| Log Analytics workspace                         | log-myenv             |
+| Azure Database for PostgreSQL flexible   server | psql-myenv            |
+| Storage account                                 | stohdsimyenv          |
```

Comment:
Is it important to list all these? Perhaps we just want to direct the user to the Atlas WebApp?

Reply:
removed